File: basex-api/src/main/scala/createexample.scala
import java.io._

/**
 * This example shows how new databases can be created.
 * Documentation: http://docs.basex.org/wiki/Clients
 *
 * @author BaseX Team 2005-12, BSD License
 */
object createexample {
  /**
   * Main method.
   * @param args command-line arguments
   */
  def main(args: Array[String]) {
    // create session
    val session = new BaseXClient("localhost", 1984, "admin", "admin")

    // define input stream
    val bais = new ByteArrayInputStream("<xml>Hello World!</xml>".getBytes)

    // create new database
    session.create("database", bais)
    println(session.info)

    // run query on database
    println(session.execute("xquery doc('database')"))

    // drop database
    session.execute("drop db database")

    // close session
    session.close
  }
}

File: basex-api/src/main/python/CreateExample.py
# -*- coding: utf-8 -*-
# This example shows how new databases can be created.
#
# Documentation: https://docs.basex.org/wiki/Clients
#
# (C) BaseX Team 2005-12, BSD License

from BaseXClient import BaseXClient

# create session
session = BaseXClient.Session('localhost', 1984, 'admin', 'admin')

try:
    # create new database
    session.create("database", "<x>Hello World!</x>")
    print(session.info())

    # run query on database
    print("\n" + session.execute("xquery doc('database')"))

    # drop database
    session.execute("drop db database")
    print(session.info())

finally:
    # close session
    if session:
        session.close()
